testing-guidance/workflow-tests/physics/physics-ragdoll-workflows.md

```diff
@@ -0,0 +1,65 @@
imation Editor layout is set to AnimGraph. Layout can be changed by expanding the dropdown menu in the top right corner of the Animation Editor.</li><li> Go to File → Open Actor and select your actor file (for example *Rin.fbx*).</li> <li>Add your actor's Motion Set (for example for *Rin.fbx* it is *MotionMatching.motionset*) by going to Motion Sets → Motion Set Management, clicking the *Load motion set from file* button and choosing the .motionset file.</li> <li>In the Anim Graph tab create a new Anim Graph by clicking the *+* button (*Create a new anim graph*) in the pane's top left corner.</li><li>Add a *Motion* node to the Anim Graph by doing **Right-click on the Anim Graph → Create Node → Sources → Motion** or by dragging it from **View → Palette Window → Sources → Motion**.</li><li>Select the created *Motion* node and view it in the *Inspector* pane. Change its *Name* (for example to *Run*). Choose a motion by clicking the *+* button (*Add motions*) and selecting one of the motions available in your actor's Motion Set (for example on *MotionMatching.motionset* choose *run1*).</li><li>Add a *Blend Tree* node to the Anim Graph by doing **Right-click on the Anim Graph → Create Node → Sources → Blend Tree** or by dragging it from **View → Palette Window → Sources → Blend Tree**.</li><li>Select the created *Blend Tree* node, and view it in the *Inspector* pane. Change its *Name* (for example to *Ragdoll*).</li><li>Connect the *Motion* node to the *Blend Tree* node by **Left-clicking the *Motion* node, holding the mouse button, and dragging it over the *Blend Tree* node**.</li><li>Select the created transition arrow and in the *Inspector* pane add a *Time Condition* by **clicking the *Add condition* button, and selecting *Time Condition* from the dropdown menu**.</li><li>Double click the *Blend Tree* node to open it and add an *Activate Ragdoll Joints* node by doing *Right-click on the graph → Create Node → Physics → Activate Ragdoll Joints* or by dragging it from **View → Palette Window → Physics → Activate Ragdoll Joints**. </li><li>Connect the *Output Pose* from the *Activate Ragdoll Joints* node to the *Input Pose* of the *Final Node*.</li><li>Select the *Activate Ragdoll Joints* and add your actor's joints by clicking the *branch* button next to the *Simulated Joints* parameter (for *Rin.fbx* add all joints present). </li><li>Save the created Anim Graph by going to **File → Save All** or pressing *Ctrl+S* shortcut.</li> | <ul><ul><li>Animation Editor launches.</li> <li>Your actor file is avaliable for selection.</li><li>New Anim Graphs can be created.</li><li>Nodes can be added to Anim Graphs.</li><li>Properties of the nodes in the Anim Graph can be adjusted.</li><li>Nodes can be connected in the Anim Graph. Example of two nodes connected:</li></br>![](images/motion-blend_tree-nodes.png)<br><li>Transition arrows can be modified in the *Inspector*.</li><li>Nodes can be added in the *Blend Trees*.</li><li>Anim Graphs can be saved.</li></ul>  |
```

Comment:
The list formatting is off for this when viewed in Rich Diff mode on the PR, but renders correctly when viewing the rendered file outside the diff. This mean that's there's a chance GH or another render tool could have it's formatting broken elsewhere.

It's likely due to your unordered lists being outside the ordered list's last list item.

You have

`<ol><li>Open the O3DE Editor.</li> <li> Open the Animation Editor (using varying entry points):</li><ul><li>O3DE Toolbar → Tools → Animation Editor</li><li>Actor Component's Actor asset property field.</li></ul>`

However, if you structured it like so it can help repair the structure.

`<ol><li>Open the O3DE Editor.</li> <li> Open the Animation Editor (using varying entry points):<ul><li>O3DE Toolbar → Tools → Animation Editor</li><li>Actor Component's Actor asset property field.</li></ul></li>
`

This formatting issue is actually present throughout this entire file, so you can fix all of the issues by following this above pattern.

Additionally: You haven't closed your ordered list at the end of the Requests column.

Lastly: On the things to watch out for column, you also open with two unordered lists which is causing the entire column to be the secondary list item.

Reply:
Fixed the formatting  for unordered lists, closed the ordered list in the Requests column, and fixed the double unordered list opening in the "Things to Watch For" column